src/arch/sparc/include/asm/ptrace.h

```diff
@@ -29,6 +29,12 @@ struct pt_regs {
 	uint32_t ins[8];     /**< Input registers. */
 }__attribute__ ((aligned (8)));
 
+#define PT_REGS_ALIGN    8
```

Comment:
Unfortunately yes!
Some architectures (SPARC for example) required stack alignment more than sizeof(void *) when we work with pt_regs we simulate work with stack